src/main/java/org/kkywalk2/LottoMachine.java

```diff
@@ -0,0 +1,42 @@
+package org.kkywalk2;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class LottoMachine{
+
+    int coinMoney;
+    int countPurchase;
+    static final int EACH_PRICE = 1000;
+
+    ArrayList<LottoNumber> lottoNumberlist = new ArrayList<LottoNumber>();
+    
+    public LottoMachine (){
+
+    }
+
+    public void purchaseLotto()
+    {
+        System.out.println("구입금액을 입력해 주세요.");
```

Comment:
가능하면 입력과 기능은 분리하는 것이 좋을 것 같아요. 특정 기능이 입출력에 종속되는 것은 좋은 구조는 아닌 것 같습니다.

Reply:
그렇다면 외부에서 입력을 받은뒤, purchaseLotto 함수에 넘겨서 처리하는 방식은 어떻게 생각하시는 지요?

Reply:
대략 어떤 방식으로 할지 코드로 보여주는 것도 좋을 것 같습니다. 다만 이견이 없는 경우 굳이 reply달지 않고 바로 적용해주셔도 좋을 것 같습니다. 아래와 같이 코멘트에 코드를 달 수도 있습니다
```suggestion
        // 이 친구를 어디에 두면 좋을까요?
        // System.out.println("구입금액을 입력해 주세요.");
```

Reply:
수정함